airflow/jobs.py

```diff
@@ -892,6 +891,11 @@ def create_dag_run(self, dag, session=None):
             if next_run_date and min_task_end_date and next_run_date > min_task_end_date:
                 return
 
+            # Don't really schedule the job, we are interested in its next run date
+            # as calculated by the scheduler
+            if dry_run is True:
+                return next_run_date
```

Comment:
@ultrabug _This_ was my complaint - I don't think we should be calling this method to work out when the next exec date is -- cos this can return None when the dag concurrecy is at it's limit for instance (line 789) and that is the problem. I think it's going to cause confusion that the "scheduled in" column would come-and-go depending on what other runs might be happening.

Reply:
ok @ashb I guess we can work on something to make it better indeed. Your point is not depending on the concurrency limit if I'm correct and display the *theorically due schedule in time". Am I right?

Reply:
Yes

Reply:
ping @ashb and @ultrabug . Can I help you guys to get this over the line?